bigbluebutton/greenlight
# Add Header

## Description

Add first iteration of Header.
Replace the CurrentUser component to see if User is signed in or not.

If User is signed in, User will see his name on top right corner, with dropdown menu.

If User is signed out, User will see a Sign In and a Sign Up button.


## Testing Steps

Pull test and make sure everything works as before.
Try in incognito when User is signed off.

## Screenshots (if appropriate):

1. When User is signed in:

![image](https://user-images.githubusercontent.com/43917914/161340941-0028c785-06c9-4e9f-9382-d1787d60a2bf.png)

2. When User is signed out:

![image](https://user-images.githubusercontent.com/43917914/161341042-5c2c7f3b-debf-4312-a5b3-f24bac51192a.png)
